controllers/nodehealthcheck_controller.go

```diff
@@ -496,8 +542,23 @@ func (r *NodeHealthCheckReconciler) patchStatus(nhc, nhcOrig *remediationv1alpha
 	return r.Client.Status().Patch(context.Background(), nhc, mergeFrom, &client.PatchOptions{})
 }
 
-func (r *NodeHealthCheckReconciler) getInflightRemediations(nhc *remediationv1alpha1.NodeHealthCheck, template *unstructured.Unstructured) (map[string]metav1.Time, error) {
-	cr, err := r.generateRemediationCR(&v1.Node{}, nhc, template)
+func (r *NodeHealthCheckReconciler) getOwnedInflightRemediations(nhc *remediationv1alpha1.NodeHealthCheck, template *unstructured.Unstructured) (map[string]metav1.Time, error) {
+	all, err := r.getAllInflightRemediations(template)
+	if err != nil {
+		return nil, err
+	}
+	owned := make(map[string]metav1.Time)
+	for _, remediationCR := range all {
+		if isOwner(&remediationCR, nhc) {
+			owned[remediationCR.GetName()] = remediationCR.GetCreationTimestamp()
+			continue
```

Comment:
NIT: Why `continue` is needed here? IIUC the logic of `getOwnedInflightRemediations` would stay the same with or without it.
There is no code after the if clause, lines 552-555, that `continue` is helping us to skip.

Reply:
indeed  👍🏼  (wasn't changed with this PR but I will remove it)